src/EPR.Payment.Mopup.Common.UnitTests/TestHelpers/TestHelperDbAsyncEnumerable.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System.Data.Entity.Infrastructure;
+using System.Diagnostics.CodeAnalysis;
+using System.Linq.Expressions;
+
+namespace EPR.Payment.Mopup.Common.UnitTests.TestHelpers
+{
+    [ExcludeFromCodeCoverage]
```

Comment:
This should not be needed in test projects as Sonar should not cover test projects for code coverage 
If this is the practice then please add justification

Reply:
we are using it for only some other unit tests methods. Not necessary to write unit test for it and if we delete the [ExcludeFromCodeCoverage], the test appears to be not covered even thought it is in unit test.

Reply:
Nothing wrong with your PR its the snar config if this class is being checked by Sonar, happy to resolve this but sonar needs to be updated to avoid coverage on test projects